Library/Homebrew/formula_installer.rb

```diff
@@ -704,6 +705,7 @@ def sanitized_argv_options
     args << "--cc=#{ARGV.cc}" if ARGV.cc
     args << "--default-fortran-flags" if ARGV.include? "--default-fortran-flags"
     args << "--keep-tmp" if Homebrew.args.keep_tmp?
+    args << "--download-silently" if fetched?
```

Comment:
I think it'd be preferable to pass this through to calls of `downloader.fetch` instead.